.github/workflows/test.yml

```diff
@@ -66,10 +66,25 @@ jobs:
 
       # use nupkg artifact for integration tests instead of project references
       - run: dotnet nuget add source "$(pwd)/artifacts"
-      - run: dotnet remove ./test/Riok.Mapperly.IntegrationTests reference ./src/Riok.Mapperly/Riok.Mapperly.csproj ./src/Riok.Mapperly.Abstractions/Riok.Mapperly.Abstractions.csproj
-      - run: dotnet add package Riok.Mapperly --version "0.0.1-dev.$GITHUB_RUN_ID" --no-restore
-        working-directory: ./test/Riok.Mapperly.IntegrationTests
-      - run: dotnet build -f net${{ matrix.dotnet }} -p:MapperlyIntegrationTestsTargetFramework=net${{ matrix.dotnet }}
+      - run: dotnet build -f net${{ matrix.dotnet }} -p:MapperlyIntegrationTestsTargetFramework=net${{ matrix.dotnet }} -p:NugetPackageVersion=0.0.1-dev.$GITHUB_RUN_ID
```

Comment:
I would prefix `NugetPackageVersion` with `Mapperly` as we do with the target framework.

Reply:
Changed